.circleci/config.yml

```diff
@@ -0,0 +1,29 @@
+# 
+version: 2
+jobs:
+  build:
+    working_directory: /build
+    docker:
+      - image: docker:17.06.0-ce-git
+    steps:
+      - checkout
+      - setup_remote_docker:
+          reusable: true
+          exclusive: true
+      - run:
+          name: Install bash
+          command: apk add --no-cache bash
+      - run:
+          name: Build image
+          command: bin/build.sh
+      - deploy:
+          name: Deploy image to Docker Hub
+          branches:
+            only:
+              - feature/circleci
```

Comment:
I plan to replace `feature/circleci` with `master` before merging and deleting `feature/circleci` branch.

I've yet to confirm adding a git tag (GitHub release). I'd hoped to do this already, but I'm blocked by CircleCI infra issue. I'll try again tonight. I thought it was a recent commit that broke it, but rebuild of previously successful builds are failing too...